rudder-web/src/main/scala/com/normation/rudder/web/comet/AsyncDeployment.scala

```diff
@@ -132,6 +132,12 @@ class AsyncDeployment extends CometActor with CometListener with Loggable {
       <li><div id="deployment-end">{displayTime("Ended at ", end)}</div></li>
       <li class="dropdown-header" style="border-bottom-style: dotted; border-color: gray;">{durationText} {DateFormaterService.getFormatedPeriod(start,end)}</li>
     }
+    def loadingStatement(start : DateTime) = {
+      <li class="dropdown-header">Policies building...</li>
+      <li>{displayTime("Started at ", start)}</li>
```

Comment:
I believe we can't use the "start" method here, because with it, we have "Started at blablabla -> 1s ago". But the "1s ago" never changes, and so it is extremelly misleading. 
So we need to only display the part with the date.